proto/babylon/incentive/events.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package babylon.incentive;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/babylonlabs-io/babylon/v2/x/incentive/types";
+
+// EventsPowerUpdateAtHeight stores an slice of events that happened at
+// some specific block height, it is indexed by the block height.
+message EventsPowerUpdateAtHeight {
+  repeated EventPowerUpdate events = 1;
```

Comment:
I guess the events are pretty small, so we could store even 1M of them at each height and not get OOM error when retrieving them from db?

Reply:
I believe so, but I didn't make any stress tests on this
The thing is, we can't limit to handle a portion each block, otherwise we would be distributing the rewards not according to the Babylon block height again. But we could limit the amount by using an iterator if you prefer, then the store would look a bit different

`(blkHeight, idx) => event`

Reply:
If events are small enough lets keep it as it is, iterators are slow as hell and should only be used as last resort (if we are sure data won't fit into memory)